ci/infra/testrunner/vars.yaml

```diff
@@ -13,13 +13,16 @@ skuba:
 # platform settings
 terraform:
   internal_net: "" # name of the internal network
-# disable mirrors
-# mirror: "ibs-mirror.prv.suse.net" # mirror url for downloading packages
   plugin_dir: "" # path to the terraform plugin dir
   stack_name: "" # name of the stack
   tfdir: ""  # path to terraform templates
   tfvars: "" # terraform vars
 
+packages:
+# disable mirrors
+# mirror: "ibs-mirror.prv.suse.net" # mirror url for downloading packages
```

Comment:
Not for this PR but maybe we should look at enabling the provo mirror again.

Reply:
I will prefer to make it in another PR to avoid this "side effect" not related to the objective of this PR.